examples/independent_control_demo.py

```diff
@@ -0,0 +1,64 @@
+"""Example file to demonstrate how to use PyDSS controllers"""
+
+import opendssdirect as dss
+
+from PyDSS.controllers import (
+    CircuitElementController,
+    ControllerManager,
+    PvControllerModel,
+)
+from PyDSS.simulation_input_models import ProjectModel
+
+
+def main():
+    settings = ProjectModel(
+        max_control_iterations=50,
+        error_tolerance=0.0001,
+    )
+    dss.run_command("compile opendss_models/Master.dss")
+    dss.Solution.Mode(0)
+    dss.utils.run_command("Set ControlMode={}".format(settings.control_mode.value))
+    dss.Solution.MaxControlIterations(settings.max_control_iterations)
+
+    volt_var_model = PvControllerModel(
+        Control1="VVar",
+        Control2="None",
+        Control3="None",
+        pf=1,
+        pfMin=0.8,
+        pfMax=1,
+        Pmin=0,
+        Pmax=1,
+        uMin=0.9399999999999999,
+        uDbMin=0.97,
+        uDbMax=1.03,
+        uMax=1.06,
+        QlimPU=0.44,
+        PFlim=0.9,
+        enable_pf_limit=False,
+        uMinC=1.06,
+        uMaxC=1.1,
+        PminVW=10,
+        VWtype="Rated Power",
+        percent_p_cutin=10,
+        percent_p_cutout=10,
+        Efficiency=100,
+        Priority="Var",
+        DampCoef=0.8,
+    )
+    controller = CircuitElementController(volt_var_model)  # Use all elements.
+    manager = ControllerManager.create([controller], settings)
```

Comment:
There is at least one problem with this design. This will create a PyDSS `abstract_solver` object which wants to control simulation time. A couple of the PyDSS controllers rely on determining and tracking simulation time, which could be different than what the user expects. There are potential solutions that we could discuss.

Reply:
1) I want to confirm my understanding of the difference between CircuitElementController and ControllerManager

CircuitElementController class: Contains the final list of elements for which the controller is to be used. Also contains the controller type information.

ControllerManager class: Assigns the controller type to each of the elements. has the ability to run controls, update controls.

2) What's the difference between running a control and updating a control?

3) I do not follow the problem that you mentioned about that design - maybe we can discuss it on our call.

Reply:
In general, the usage looks simple and easy!